src/plugins/databases/index.js

```diff
@@ -940,6 +945,10 @@ export default class DatabasesPlugin extends FlipperPlugin<
         databases,
       });
     });
+    this.dispatchAction({
+      type: 'UpdateFavorites',
+      favorites: JSON.parse(localStorage.getItem('favoritesSQLQueries')),
```

Comment:
There are a couple of flow errors which somehow aren't shown in CI (will investigate that separately).

This may fail because `localStorage.getItem` can return undefined. I'd just turn this into `... || '[]'`.